packages/ncurses/build.sh

```diff
@@ -20,7 +20,7 @@ TERMUX_PKG_VERSION=(6.5.20240831
 TERMUX_PKG_REVISION=2
```

Comment:
You'll need to bump the revision or commit this with a `[no ci]` tag.

Reply:
Done